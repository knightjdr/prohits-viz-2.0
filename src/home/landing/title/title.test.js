import React from 'react';
import { shallow } from 'enzyme';

import Title from './title';

describe('Title', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <Title />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
