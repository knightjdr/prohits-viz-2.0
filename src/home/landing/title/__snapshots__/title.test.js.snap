// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should match snapshot 1`] = `
<h1
  className="landing__title"
>
  ProHits-viz
  <img
    alt="ProHits-viz"
    src="title.svg"
  />
</h1>
`;
